Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM dockerfile/nodejs
+FROM node:latest
```

Comment:
You are right. Do you have any recommendations for the version that should be used?